src/view/seed-verify.js

```diff
@@ -41,7 +40,7 @@ const SeedVerifyView = ({ store, nav, wallet }) => (
     </Header>
     <MainContent style={styles.content}>
       <View>
-        <H1Text style={styles.title}>{"Let's double check"}</H1Text>
+        <H1Text style={styles.title}>Restore your wallet</H1Text>
```

Comment:
Are you sure this is what we want in the seed-verify screen? Seems like the copy the way it was is ok.

Reply:
My mistake, fixed